Example/android/app/src/main/res/values/styles.xml

```diff
@@ -3,6 +3,7 @@
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <!-- Customize your theme here. -->
+        <item name="android:windowTranslucentStatus">true</item>
```

Comment:
What does it do?

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/827/commits/f9631c5e3f89dae24f2aef61883b0e53da8c27a7